tests/data_files/server1.ca.crt

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Not a blocker: We try to add build instructions for new test data in `tests/data_files/Makefile`. Could you add two rules building the two new certificates using `programs/x509/cert_write`?

Reply:
Done